### ISSUE TITLE
Dataset selection for training and test

### ISSUE THREAD

**Issue by hellocasper** [Liumouliu/deepIBL#1]
Hi Liuliu,
Thanks for your amazing job. 
As stated in the paper, you took pitts30k-train as the training dataset and others for the test.
But in the sample script, the code shows that pitts250k-train is taken as training dataset. Which one  should we follow to reproduce the results? Or the model trained with pitts250k is used for classification?
Another question is that: are the results in table 2 of the paper gotten after implementing pca whitening with dim 4096?

**Comment by Liumouliu** [comment]
Hi, 
    Thank you for your interest.
    Yes, pitts30k-train is the only training set other than the classification net. You notice that pitts250k-train is in the sample script. However, it is switched off. (doPitts250k= false;) Thus, pitts30k-train is activated.
    Yes, after PCA.

Best,
Liu
